app/viewModels/houseConsignment/index/items/NumberOfPackagesViewModel.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewModels.houseConsignment.index.items
+
+import models.{CheckMode, Index, Mode, NormalMode}
+import play.api.i18n.Messages
+
+case object NumberOfPackagesViewModel {
```

Comment:
The idea of a view model is that we pass the view model into the view (classic MVVM pattern)

So ideally this would be a case class containing all of the "stuff" we want in the view. Then a companion object with an apply method that takes the `mode`, `houseConsignmentIndex` and `itemIndex`.

In terms of testability, what we've often done in the past is include a provider class which we inject into the controller. I'll dig out an example.

https://github.com/hmrc/ctc-departure-transport-details-frontend/blob/main/app/viewModels/transportMeans/departure/MeansIdentificationNumberViewModel.scala